sdk/storage/azure-storage-file-datalake/samples/datalake_samples_file_system.py

```diff
@@ -209,6 +209,49 @@ def create_file_from_file_system(self):
 
         file_system_client.delete_file_system()
 
+        # [START batch_delete_files_or_empty_directories]
```

Comment:
nit: Move this back a tab? While this may feel really nit, there is a chance this is needed for the documentation to correctly link to the sample.

Same for async sample.